docs/section22/Rule22-20.md

```diff
@@ -0,0 +1,40 @@
+
+# CHW&CW - Rule 22-20  
+
+**Rule ID:** 22-20  
+**Rule Description:** The baseline heat rejection Loop temperature reset = Load Reset.  
```

Comment:
G3.1.3.11 that this RDS is meant to implement has requirements quoted below. I believe the requirements represent reset based on wet bulb temperature, not based on load. Let's discuss and perhaps consider adding wet-bulb temperature reset option to the schema. 

Condenser-water design supply temperature shall be calculated using the cooling tower approach to the 0.4% evaporation design wet-bulb temperature as generated by the formula below, with a design temperature rise of 10°F:
Approach10°F Range = 25.72 – (0.24 × WB) 
where WB is the 0.4% evaporation design wet-bulb temperature (°F); valid for wet bulbs from 55°F to 90°F.

Reply:
@mkarpman I might be misunderstanding this requirement, but I don't read this as a wet bulb reset either. we should discuss this

Reply:
I quoted a wrong section. I think the relevant rule is below. I don't think this is LOAD_RESET, but no other TemperatureResetTypeOptions seem to fit. Perhaps "Other"? 

The tower shall be controlled to maintain a leaving water temperature, where
weather permits, per Table G3.1.3.11, floating up to the design leaving water temperature
for the cooling tower.

Reply:
Updated.